code/modules/battlemonsters/datum_elements.dm

```diff
@@ -194,22 +219,56 @@
 	name = "Igloo Defender"
 	id = "igloo"
 	icon_state = "igloo"
-	description = "Behind them is a giant igloo."
-	special_effects = "Master Defender: As long as %NAME is visible, it's owner cannot be attacked by monsters. "
+	description = "They stand silently outside an igloo in a freezing blizzard with a sturdy grip on their %WEAPON_AND. They seem unbothered by the cold."
+	attack_type = BATTLE_MONSTERS_ATTACKTYPE_SHIELD
+	power_add = 200
+	special_effects = "Master Defender: As long as %NAME is visible, its owner cannot be attacked by monsters. "
 	rarity = BATTLE_MONSTERS_RARITY_UNCOMMON
 
 /datum/battle_monsters/element/ice/yeti
 	name = "Yeti"
 	id = "yeti"
 	icon_state = "white"
-	description = "They're covered head to toe in white, absorbant magical fur."
+	description = "They're covered head to toe in white, absorbant magical fur. Their %WEAPON_AND is ferocious, especially for a %SPECIES."
 	special_effects = "Spell Absorbtion: As long as %NAME is visible, it's owner cannot be affected by traps or spells."
 	defense_type = BATTLE_MONSTERS_DEFENSETYPE_CREATURE
 	attack_type = BATTLE_MONSTERS_ATTACKTYPE_CLAWS
 	power_add = 200
 	rarity = BATTLE_MONSTERS_RARITY_RARE
 	rarity_score = 1
 
+/datum/battle_monsters/element/ice/frostlord
+	name = "Frostlord"
+	id = "frostlord"
+	icon_state = "ice"
+	description = "They're wearing a large crown made out of ice and diamonds, and a royal cape made out magical yeti fur. Frostlords gain their title by defeating a yeti in combat by wrestling it to death."
```

Comment:
Missing word after `out`. Latter sentence seems a little awkward too, but I can't think of a way to fix it off the top of my head.